javascript/Client.js

```diff
@@ -13,7 +13,8 @@ export default class Client {
     this._componentTracker =
       new AttributeTracker(this.keyAttribute, (element) => (
         element.hasAttribute(this.stateAttribute) // ensure matches selector
-          ? new Component(this, element) : null
+          ? new Component(this, element)
+          : null
```

Comment:
I decided to update all JS dependencies as a part of this change which included an update to Standard which required this small formatting change.